sdk/resourcemanager/Azure.ResourceManager.Core/docs/Prerequisites.md

```diff
@@ -0,0 +1,55 @@
+Prerequisites
+-------------
+
+You will need the following values to authenticate to Azure
+
+-   **Subscription ID**
+-   **Client ID**
+-   **Client Secret**
+-   **Tenant ID**
+
+These values can be obtained from the portal, here's the instructions:
+
+### Get Subscription ID
+
+1.  Log in to your Azure account.
+2.  Select **Subscriptions** in the left sidebar.
+3.  Select the subscription to be used.
+4.  Select **Overview**.
+5.  Copy the Subscription ID.
+
+### Get Client ID / Client Secret / Tenant ID
+
+For information on how to get Client ID, Client Secret, and Tenant ID, see [this
+document](https://docs.microsoft.com/azure/active-directory/develop/howto-create-service-principal-portal).
+
+### Setting Environment Variables
+
+After you obtain the values, set the following environment variables:
+
+-   `AZURE_CLIENT_ID`
+-   `AZURE_CLIENT_SECRET`
+-   `AZURE_TENANT_ID`
+-   `AZURE_SUBSCRIPTION_ID`
+
+To set the environment variables on your development system:
+
+Windows:
+
+_(Note: Administrator access is required)_
+
```

Comment:
these are not requirements.  This is elevating the SPN + secret credential as if it is the primary credential everyone should use.  It is not.  SPN + Secret is a very specialized credential used fro some kinds of automation.  It would not be the most used credential type